frontend/src/components/controls/RisDateInput.vue

```diff
@@ -117,17 +156,48 @@ watch(inputCompleted, (is) => {
 </script>
 
 <template>
-  <input
-    :id="id"
-    v-model="inputValue"
-    v-maska
-    class="ds-input"
-    :class="conditionalClasses"
-    data-maska="##.##.####"
-    placeholder="TT.MM.JJJJ"
-    :readonly="isReadOnly"
-    @blur="onBlur"
-    @keydown.delete="backspaceDelete"
-    @maska="onMaska"
-  />
+  <div
+    :class="
+      props.labelPosition === 'left'
+        ? `|| 'gap-8'} flex items-center gap-4`
+        : 'grid gap-2'
+    "
+  >
+    <!-- eslint-disable-next-line vuejs-accessibility/label-has-for -->
```

Comment:
can we nest the input within this label? That should solve this warning.

Reply:
Good point